CHANGELOG.md

```diff
@@ -16,6 +24,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Fix usage on module singleton classes
 - Fix usage on module which would be extended by other classes
+### Breaking Changes
```

Comment:
Should we add these sections for past sections too?

Reply:
I thought about it. I decided not to because I figured that we only publicized the gem after the 1.0.0 release, so it was unlikely people were upgrading from versions before that.

Reply:
Fine by me. I don't think I see a reason not to add them but I don't feel strongly; either way works.